apps/estuary/config/integration.exs

```diff
@@ -1,5 +1,7 @@
 use Mix.Config
 
+endpoints = [localhost: 9092]
+
 config :prestige,
   base_url: "http://127.0.0.1:8080",
   headers: [
```

Comment:
Can we also change this schema to not be hive.default

Reply:
What would you like me to do instead?